legion_airflow/legion_airflow/hooks/s3_hook.py

```diff
@@ -507,12 +521,8 @@ def _format_row(cells: list, column_splitter: str = ',', quote: str = '"'):
         row = ''
         for cell in cells:
             if column_splitter in cell or quote in cell or '\n' in cell:  # if cell contains comma or quote
-                protected_cell = quote + cell.replace(quote, quote + quote) + quote  # wrap cell in quotes
```

Comment:
Could you please comment this change?

Reply:
This change was done due to code refactoring.